exir/lowered_backend_module.py

```diff
@@ -720,6 +721,11 @@ def create_exported_program_from_submodule(
     in_spec = pytree.tree_flatten((tuple(subgraph_signature.user_inputs), {}))[1]
```

Comment:
todo: add a check for examing the input signature of first partition and original edge program